packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -228,6 +228,11 @@ class Popover extends React.Component {
       }
     }
 
+    // Handle full-width menu
+    if (anchorPosition.horizontal === "full-width") {
+      left = 0;
+    }
+
     this.setState(
```

Comment:
I would consider making a fullWidth prop, and the if that is set doing a separate calculation at the beginning of this function. That way the logic is separated and it would be easy to remove later if we can fix the issue